friends/src/Form.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import './App.css'
+import axios from 'axios'
+class Form extends React.Component{
+    constructor(props){
+        super(props);
+        this.state ={
+            friends: [],
+            nameInput: '',
+            ageInput: null,
+            emailInput:''
+        }
+    }
+    componentDidMount() {
+        axios.get('http://localhost:5000/friends')
+            .then((response) => {
+                this.setState(() => ({ friends: response.data }));
+             })
+            .catch(error => {
+                console.error('Server Error', error);
+            });
+    }
+    nameHandler = event =>{
+        this.setState({
+            nameInput: event.target.value,
+        })
+    }
+    ageHandler = event => {
+        this.setState({
+            ageInput: event.target.value,
+        })
+    }
+    emailHandler = event => {
+        this.setState({
+            emailInput: event.target.value,
+        })
+    }
+    postFriend = (friends) => {
+        axios.post('http://localhost:5000/friends', friends)
+        .then(response => console.log(response))
+        .catch(err => console.log(err))
+    }
+    addFriend = event =>{
+        this.postFriend({'name':`${this.state.nameInput}`,'age':`${this.state.ageInput}`, 'email':`${this.state.emailInput}`});
```

Comment:
you could have had you state like 
```
this.state = {
            name: '',
            age: 0,
            email: '',
        }
```
this it could have avoided some extra work you did.